app/views/content/life-as-a-teacher/explore-subjects/music/_article.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach music</h2>
+      <p> Through teaching music, you will share your passion and encourage pupils to discover theirs. From composing and performing to being a critical listener, you'll give pupils the chance to have fun and develop their appreciation for different types of music.</p>
+      <%= render 'content/shared/teaching/benefits' %>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching music opens the door for your pupils to pursue exciting careers. From music producers and sound engineers, to musical artists and composers, you may just inspire the next big thing.</p>
+      <p>Your pupils will learn more from you than you might think. As their music teacher, you 'll guide your pupils through the skills they need to develop their appreciation of music and how it's constructed. You'll teach them valuable life skills such as reflective listening, how to make considered choices and the confidence to take risks.</p>
+    </div>
+   </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-music-programmes-of-study/national-curriculum-in-england-music-programmes-of-study">national curriculum for music</a>.</p>
+    <p>Throughout key stage 3 pupils will build on their skills through performing, composing and listening. Pupils will be taught to: </p>
+    <ul>
+      <li>perform confidently</li>
+      <li>improvise and compose musical ideas </li>
+      <li>use staff and other notations accurately </li>
+      <li> listen to a wide range of composers and musicians</li>
+      <li>develop a deeper understanding of music and its history</li>
+    </ul>
+    <p>In key stage 4 (14 to 16 years olds) you will help pupils demonstrate skills in:</p>
+    <ul>
+      <li>developing composing </li>
+      <li>recognising contrasting genres, styles and traditions</li>
+      <li>use staff and other notations accurately </li>
+      <li>reading and writing of staff notation</li>
+      <li>analysing and evaluating music</li>
+      <li>making critical judgements about music</li>
+    </ul>
+    <p>Explore what teaching a key stage 3 and 4 music lesson would be like at the <a href="https://www.thenational.academy/teachers/curriculum/music-secondary-edexcel/units">Oak National Academy website</a>.</p>
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop a deeper knowledge and understanding of:</p>
+    <ul>
+      <li>performance</li>
+      <li>composing</li>
+      <li>awareness of music</li>
```

Comment:
This bullet doesn't quite fit with the others - does it need rewording slightly?